backendv2/pkg/tofu/config.go

```diff
@@ -0,0 +1,126 @@
+type providerConfig struct {
+	Name              string         `json:"name,omitempty"`
+	FullName          string         `json:"full_name,omitempty"`
+	Alias             string         `json:"alias,omitempty"`
+	VersionConstraint string         `json:"version_constraint,omitempty"`
+	ModuleAddress     string         `json:"module_address,omitempty"`
+	Expressions       map[string]any `json:"expressions,omitempty"`
+}
+
+type module struct {
+	Outputs map[string]output `json:"outputs,omitempty"`
+	// Resources are sorted in a user-friendly order that is undefined at this
+	// time, but consistent.
+	Resources   []resource            `json:"resources,omitempty"`
+	ModuleCalls map[string]moduleCall `json:"module_calls,omitempty"`
+	Variables   variables             `json:"variables,omitempty"`
+}
+
+type moduleCall struct {
+	Source            string         `json:"source,omitempty"`
+	Expressions       map[string]any `json:"expressions,omitempty"`
+	CountExpression   *expression    `json:"count_expression,omitempty"`
+	ForEachExpression *expression    `json:"for_each_expression,omitempty"`
+	Module            *module        `json:"module,omitempty"`
+	VersionConstraint string         `json:"version_constraint,omitempty"`
+	DependsOn         []string       `json:"depends_on,omitempty"`
+}
+
+// variables is the JSON representation of the variables provided to the current
+// plan.
+type variables map[string]*variable
+
+type variable struct {
+	Type        json.RawMessage `json:"type,omitempty"`
+	Default     json.RawMessage `json:"default,omitempty"`
+	Description string          `json:"description,omitempty"`
+	Required    bool            `json:"required,omitempty"`
+	Sensitive   bool            `json:"sensitive,omitempty"`
+	Ephemeral   bool            `json:"ephemeral,omitempty"`
+	Deprecated  string          `json:"deprecated,omitempty"`
+}
+
+// Resource is the representation of a resource in the config
+type resource struct {
+	// Address is the absolute resource address
+	Address string `json:"address,omitempty"`
+
+	// Mode can be "managed" or "data"
+	Mode string `json:"mode,omitempty"`
+
+	Type string `json:"type,omitempty"`
+	Name string `json:"name,omitempty"`
+
+	// ProviderConfigKey is the key into "provider_configs" (shown above) for
+	// the provider configuration that this resource is associated with.
+	//
+	// NOTE: If a given resource is in a ModuleCall, and the provider was
+	// configured outside of the module (in a higher level configuration file),
+	// the ProviderConfigKey will not match a key in the ProviderConfigs map.
+	ProviderConfigKey string `json:"provider_config_key,omitempty"`
+
+	// Provisioners is an optional field which describes any provisioners.
+	// Connection info will not be included here.
+	Provisioners []provisioner `json:"provisioners,omitempty"`
+
+	// Expressions" describes the resource-type-specific  content of the
+	// configuration block.
+	Expressions map[string]any `json:"expressions,omitempty"`
```

Comment:
```suggestion
	Expressions map[string]expression `json:"expressions,omitempty"`
```

Shouldn't be this one?